unlock-app/src/components/interface/locks/List/elements/LockList.tsx

```diff
@@ -17,7 +17,7 @@ export const NoItems = () => {
       src="/images/illustrations/no-locks.svg"
       description={
         <>
-          <span>You have not create any membership contract yet. </span>
+          <span>No Locks created.</span>
```

Comment:
Thanks for the correction! Maybe a simper edit will be  

```suggestion
          <span>You have not created any membership contracts yet.</span>
```